cni/plugin.go

```diff
@@ -177,3 +151,39 @@ func (plugin *Plugin) Error(err error) *cniTypes.Error {
 func (plugin *Plugin) Errorf(format string, args ...interface{}) *cniTypes.Error {
 	return plugin.Error(fmt.Errorf(format, args...))
 }
+
+// Initialize key-value store
+func (plugin *Plugin) InitializeKeyValueStore(config *common.PluginConfig) error {
+	// Create the key value store.
+	if plugin.Store == nil {
+		var err error
+		plugin.Store, err = store.NewJsonFileStore(platform.CNIRuntimePath + plugin.Name + ".json")
+		if err != nil {
+			log.Printf("[cni] Failed to create store, err:%v.", err)
+			return err
+		}
+
+		// Acquire store lock.
+		err = plugin.Store.Lock(true)
```

Comment:
we started following this convention now.. in old code we dint change to this..but in new code we are following this
if err := function(); err != nil { 
}

Reply:
Done